src/styles/blocks/page.scss

```diff
@@ -0,0 +1,55 @@
+.html {
+  font-family: arial, helvetica, sans-serif;
+}
+
+.body {
+  padding-top: 50px;
+}
+
+.calendar {
```

Comment:
Create a separate file per each BEM block styles that have the same name as the block

Reply:
fix it 
![image](https://user-images.githubusercontent.com/97547824/208163758-683fbd2c-f538-4d7b-a192-439d189455b0.png)